donut/templates/includes/header.html

```diff
@@ -40,10 +40,13 @@
                         <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Committees <span class="caret"></span></a>
                         <ul class="dropdown-menu" role="menu">
                             <li><a href="https://sites.google.com/site/arccaltech/">ARC</a></li>
-			    <li><a href="{{ url_for('uploads.display', url='BoC') }}">BoC</a></li>
+                            <li><a href="{{ url_for('uploads.display', url='BoC') }}">BoC</a></li>
                             <li><a href="{{ url_for('uploads.display', url='CRC') }}">CRC</a></li>
                             <li><a href="http://ihc.caltech.edu/">IHC</a></li>
                             <li><a href="#">ESC</a></li>
+                            {% if 'permissions' in session and 'username' in session and 1 == session['permissions'] %}
```

Comment:
I don't think we're using `session['permissions']` to store any values

Reply:
Hmmm, do you think it'll be a good idea to store something within session['x_permissions'] so that in the layout.html, you can check if a certain tab has to be generated? I'm not sure if there are other ways around this?

Reply:
Can't you just pass a variable when you call `render_template()` that says whether or not to show the tab?

Reply:
The header is a part of layout.html, which is being extended in every page.

Reply:
Hmm, maybe something like that would make sense, but I definitely don't think we want to be storing the permissions in the session storage. If we set `session['permissions']`, can we add `del session['permissions']` to `teardown_request()` in `donut/__init__.py`?